packages/main/src/plugin/extension-loader.ts

```diff
@@ -1457,16 +1509,20 @@ export class ExtensionLoader {
         console.log(`Activating extension (${extension.id}) ended in ${Math.round(duration)} milliseconds`);
       }
       const id = extension.id;
+      const packageJSON = extension.manifest;
       const activatedExtension: ActivatedExtension = {
         id,
+        packageJSON,
         deactivateFunction,
         extensionContext,
+        exports,
       };
       this.activatedExtensions.set(extension.id, activatedExtension);
       this.extensionState.set(extension.id, 'started');
       this.apiSender.send('extension-started');
     } catch (err) {
       console.log(`Activating extension ${extension.id} failed error:${err}`);
+      this._onDidChange.fire();
```

Comment:
Why fire an event on error ? Nothing has change as it was not completed ?

Reply:
fixed